### ISSUE TITLE
Wownero

### ISSUE THREAD

**Issue by duggavo** [retoaccess1/haveno-reto#18]
Wownero support would be a great addition to Reto.

**Comment by boldsuck** [comment]
You are completely wrong here. It is neither an issue nor is Haveno being developed in this repo.
PR: has been on ice for a year.
https://github.com/haveno-dex/haveno/pull/690

Submit new listings in the right place:
https://github.com/haveno-dex/listing

Be sure to read and fill out the template, I demonstrated it with Tari. ;-)
